api/src/pcapi/routes/public/individual_offers/v1/events.py

```diff
@@ -248,7 +248,7 @@ def edit_event(event_id: int, body: serialization.EventOfferEdition) -> serializ
                     if "categoryRelatedFields" in updates
                     else extra_data
                 ),
-                isActive=get_field(offer, updates, "isActive"),
+                isActive=get_field(offer, updates, "isActive", nullable=False),
```

Comment:
Alors je pense que l'approche qui consiste à ajouter un paramètre à `get_field` dont le fonctionnement n'est pas d'une grande clarté, est peut-être un peu trop complexe.

Je te propose de plutôt faire quelque chose comme ça:
```suggestion
                isActive=get_field(offer, updates, "isActive") or offer.isActive,
```
Je ne sais pas si mypy permet de faire ça mais tu peux faire un truc un peu similaire